docs/spec/auth/vesting.md

```diff
@@ -0,0 +1,88 @@
+## Vesting
+
+### Intro and Requirements
+
+This paper specifies changes to the auth and bank modules to implement vesting accounts for the Cosmos Hub. 
+The requirements for this vesting account is that it should be capable of being initialized during genesis with
+a starting balance X coins and a vesting blocktime T. The owner of this account should be able to delegate to validators 
+and vote with locked coins, however they cannot send locked coins to other accounts until those coins have been unlocked. 
+The vesting account should also be able to spend any coins it receives from other users or from fees/inflation rewards. 
+Thus, the bank module's `MsgSend` handler should error if a vesting account is trying to send an amount that exceeds their 
+unlocked coin amount.
```

Comment:
So yeah this is correct - however this "unlock amount" should not include fees rewards withdrawn as those should be withdrawn directly to a seperate non-vested account